docs/components/miranum-ide/miranum-modeler.md

```diff
@@ -0,0 +1,35 @@
+---
+id: miranum-modeler
+title: Miranum Modeler
+sidebar_label: Miranum Modeler  
+description: "This section contains an overview of the functionality of the Miranum Modeler."
+---
+
+> Model your BPMN 2.0 diagrams right in your IDE
+
+Miranum Modeler is one part of our Integrated Development Environment ([Miranum-IDE](intro-miranum-ide.md)).
+It's a Visual Studio Code (VS Code) plugin you can find in the [official marketplace](https://marketplace.visualstudio.com/items?itemName=miragon-gmbh.vs-code-bpmn-modeler).
+
+The plugin targets the creation of Camunda Platform 7 compatible BPMN diagrams and enhance them with technical attributes.
+It uses the Open Source Projects [bpmn-js](https://bpmn.io/toolkit/bpmn-js/) and [VS Code](https://code.visualstudio.com/) 
+to make the life of an BPM Developer a little easier.
+
+## Features
+
+### Model your BPMN 2.0 diagram
+
+![create-process.gif](@site/docs/components/miranum-ide/static/img/miranum-modeler/create-process.gif)
+
```

Comment:
💭 I would prefer to have a list of bullet points explaining what you are showing in the gif right here.

Reply:
I added the description to the gifs.